src/include/duckdb/main/extension_helper.hpp

```diff
@@ -87,6 +87,21 @@ class ExtensionHelper {
 	static string GetExtensionName(const string &extension);
 	static bool IsFullPath(const string &extension);
 
+	//! Lookup a name + type in an ExtensionFunctionEntry list
+	template <size_t N>
+	static pair<string, string> FindExtensionInFunctionEntries(const string &name,
```

Comment:
Hmm this actually doesn't work correctly, I noticed `json_execute_serialized_sql` exists both as a `pragma` and as a `table` function.
This should probably return a vector and we need to check if none of the entries match the requested function type.